### PR TITLE
docs: add notes from in person session between Lars and David

### PR DESCRIPTION
**Pull request by diasdavid**
@lgierth would you like to add anything?

### CODE REVIEW

**Comment by diasdavid** [comment]
@skylar more notes from out of band discussions :) Thank you keep reminding everyone to do this :)

**Comment by lgierth** [comment]
Thaaaaanks so much David, I'll look over it now

**Comment by diasdavid** [comment]
Thank you for the feedback, @VictorBjelkholm! Appreciated :) 

@lgierth good to merge?

File: meeting-notes/2017-08-30--coffee-lars-david.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+- [ ] Change libp2p-webrtc-star to p2p-webrtc-star (so that it matches the nomenclature also used by p2p-circuit)
+- [ ] Apply the same name change and encapsulation change to libp2p-websockets-star
+
+#### Gateways vs Boostrappers
+
+One of the promises of js-ipfs is that content added in the browser is visible though the gateways. This promise was recently broken when the Bootstraper nodes became independent from the gateway as reported here https://github.com/ipfs/js-ipfs/issues/967#issuecomment-325732398.
+
+Today there are only two gateways, chappy and scrappy.
+
+- [ ] Solve the issue by adding the gateways multiaddrs to the bootstrapers list on https://github.com/ipfs/js-ipfs/tree/master/src/core/runtime
```
Comment (diasdavid) [inline meeting-notes/2017-08-30--coffee-lars-david.md]: tracked here https://github.com/ipfs/js-ipfs/issues/973

File: meeting-notes/2017-08-30--coffee-lars-david.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+- [ ] Core Review libp2p-websockets-star
+- [ ] Work with @mkg20001 to ensure it is tested throughout the stack
+- [ ] Deploy a Rendezvous point
+
+#### (added topic) Delegated Routing
+
+This is a long time coming and with the modularity in js-libp2p, really doable.
+
+- [ ] Make the findproovs call publicly accessible throught the gateways
```
Comment (diasdavid) [inline meeting-notes/2017-08-30--coffee-lars-david.md]: @lgierth when will be able to have this?

File: meeting-notes/2017-08-30--coffee-lars-david.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+#### WebRTC Star multiaddrs situation
+
+We discussed https://github.com/libp2p/js-libp2p-webrtc-star/issues/110 in length and decided that:
+
+- [ ] libp2p-webrtc-star should encapsulate another multiaddr (as described in the issue) and that means that a `/libp2p-webrtc-star/dns4/star-signal.cloud.ipfs.team/wss/ipfs/<your-peer-id>` should in fact be `/dns4/star-signal.cloud.ipfs.team/wss/libp2p-webrtc-star/ipfs/<your-peer-id>`. This requires
+  - [ ] Updating js-mafmt
+  - [ ] Bubble up all the changes through the dep tree
+- [ ] Change libp2p-webrtc-star to p2p-webrtc-star (so that it matches the nomenclature also used by p2p-circuit)
+- [ ] Apply the same name change and encapsulation change to libp2p-websockets-star
```
Comment (diasdavid) [inline meeting-notes/2017-08-30--coffee-lars-david.md]: tracked here https://github.com/ipfs/js-ipfs/issues/981

File: meeting-notes/2017-08-30--coffee-lars-david.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+## Notes
+
+#### The inner works of WebRTC Star
+
+We went through in length of how WebRTC works from signalling, SDP offers, handshake, the crypto challenge and NAT Traversal using STUN. Lars got the information he was looking for to understand how WebRTC Star works.
+
+- [ ] [Create a full illustration and write down the same explanation](https://github.com/libp2p/js-libp2p-webrtc-star/issues/117) for the [js-libp2p-webrtc-star repo](https://github.com/libp2p/js-libp2p-webrtc-star).
+
+#### WebRTC Star multiaddrs situation
```
Comment (diasdavid) [inline meeting-notes/2017-08-30--coffee-lars-david.md]: tracked here https://github.com/ipfs/js-ipfs/issues/981

**Pull request merged**